### ISSUE TITLE
Javascript console errors navigating to project

### ISSUE THREAD

**Issue by AlanCoding** [ansible/awx#865]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
Console errors clicking on the link to edit a project

##### ENVIRONMENT
* AWX version: just rebuilt the UI on 1.0.2-301, error did not surface right before this
* AWX install method: docker for mac
* Operating System: mac
* Web Browser:Chrome

##### STEPS TO REPRODUCE
open up AWX, navigate to an existing project

##### EXPECTED RESULTS
no console errors

##### ACTUAL RESULTS
multiple console errors triggered by that single action

first

```
TypeError: $(...).spinner is not a function
    at Object.link (directives.js:1060)
    at angular.js:1383
    at invokeLinkFn (angular.js:10611)
    at nodeLinkFn (angular.js:10000)
    at compositeLinkFn (angular.js:9240)
    at compositeLinkFn (angular.js:9243)
    at nodeLinkFn (angular.js:9994)
    at compositeLinkFn (angular.js:9240)
    at nodeLinkFn (angular.js:9994)
    at compositeLinkFn (angular.js:9240) "<input value="0" class="ng-pristine ng-untouched ng-valid" aw-spinner="scm_update_cache_timeout" ng-model="scm_update_cache_timeout" name="scm_update_cache_timeout" id="scm-cache-timeout" min="0" ng-disabled="!(project_obj.summary_fields.user_capabilities.edit || canAdd)" integer="">"
```

second

```
TypeError: $(...).spinner is not a function
    at directives.js:1053
    at Scope.$digest (angular.js:18359)
    at Scope.$apply (angular.js:18628)
    at done (angular.js:12619)
    at completeRequest (angular.js:12863)
    at XMLHttpRequest.requestLoaded (angular.js:12780)
```

third, likely a repeat of the first

```
TypeError: $(...).spinner is not a function
    at directives.js:1053
    at Scope.$digest (angular.js:18359)
    at Scope.$apply (angular.js:18628)
    at done (angular.js:12619)
    at completeRequest (angular.js:12863)
    at XMLHttpRequest.requestLoaded (angular.js:12780)
```

##### ADDITIONAL INFORMATION

N/A

**Comment by AlanCoding** [comment]
also happens navigating to job template

**Comment by mabashian** [comment]
Took a quick peek - any form with a field where the attribute `spinner` is set to `true` seems to throw this error.

**Comment by mabashian** [comment]
This should be addressed by #872
